app/client/sagas/authSagas.js

```diff
@@ -1,11 +1,8 @@
 import {
```

Comment:
Definitely think we should be also converting sagas to typescript

Reply:
yeah for sure. i was going to do that in this PR too but it was getting chaotic and I'm working on keeping the changes more incremental